<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-LVLR9NPK" failures="1" tests="6" name="FactorialTest" time="86.248" errors="0" timestamp="2020-10-03T21:38:49 EDT">
  <testcase classname="com.tribalescale.qa.testCases.TC_FACT_002" name="invalidInput" time="0.752">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at com.tribalescale.qa.testCases.TC_FACT_002.invalidInput(TC_FACT_002.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- invalidInput -->
  <testcase classname="com.tribalescale.qa.testCases.TC_FACT_002" name="invalidInput" time="0.471"/>
  <testcase classname="com.tribalescale.qa.testCases.TC_FACT_002" name="invalidInput" time="0.612"/>
  <testcase classname="com.tribalescale.qa.testCases.TC_FACT_002" name="invalidInput" time="0.529"/>
  <testcase classname="com.tribalescale.qa.testCases.TC_FACT_002" name="invalidInput" time="0.556"/>
  <testcase classname="com.tribalescale.qa.testCases.TC_FACT_002" name="invalidInput" time="0.768"/>
</testsuite> <!-- FactorialTest -->
